src/core/menu_items/WifiMenu.cpp

```diff
@@ -14,6 +14,18 @@
 #include "modules/pwnagotchi/pwnagotchi.h"
 #endif
 
+#include "modules/reverseShell/reverseShell.h"
+// Developed by Fourier (github.com/9dl)
+// Use BruceC2 to interact with the reverse shell server
+// BruceC2: https://github.com/9dl/Bruce-C2
+// To use BruceC2:
+// 1. Start Reverse Shell Mode in Bruce
+// 2. Start BruceC2 and wait.
+// 3. Visit 192.168.4.1 in your browser to access the web interface for shell executing.
+
+// 32bit: https://github.com/9dl/Bruce-C2/releases/download/v1.0/BruceC2_windows_386.exe
```

Comment:
Please Add these files into a .zip in the ./sd_files/reverseshell folder and, if you may, add the BADUSB payload to this folder, so we can ,merge with all files set..